src/ctr_exit.c

```diff
@@ -133,32 +134,13 @@ void container_exit_cb(G_GNUC_UNUSED GPid pid, int status, G_GNUC_UNUSED gpointe
 	g_main_loop_quit(main_loop);
 }
 
-static void close_fd(int *fd)
-{
-	if (*fd >= 0)
-		close(*fd);
-	*fd = -1;
-}
-
 void do_exit_command()
 {
-	if (sync_pipe_fd > 0) {
-		close(sync_pipe_fd);
-		sync_pipe_fd = -1;
-	}
-
 	if (signal(SIGCHLD, SIG_DFL) == SIG_ERR) {
 		_pexit("Failed to reset signal for SIGCHLD");
 	}
 
-	close_logging_fds();
```

Comment:
Is close_loggin_fds() being called elsewhere?  If not, should it be removed?

Reply:
we are closing each fd >= 3, so these files will all be closed